src/dropdown/test/accessibility.test.ts

```diff
@@ -104,6 +104,37 @@ describe("Dropdown", () => {
       ).to.equal(false);
     });
 
+    it("should not highlight when mouseover the disabled item", async () => {
+      const container = new Dropdown({
+        items: [
+          ...initItems,
+          { label: "Apple", value: "apple", disabled: true },
+        ],
+        value: initItems[0].value,
+      });
+      const el = await fixture(container);
+      const toggleEl = el.querySelector(
+        ".kuc-dropdown__group__toggle"
+      ) as HTMLButtonElement;
+      toggleEl.click();
+      const itemsEl = el.querySelectorAll(
+        ".kuc-dropdown__group__select-menu__item"
+      );
+
+      itemsEl[3].dispatchEvent(new Event("mouseover"));
+      await expect(
+        itemsEl[3].classList.contains(
+          "kuc-dropdown__group__select-menu__highlight"
+        )
+      ).to.equal(false);
+
+      await expect(
+        itemsEl[0].classList.contains(
+          "kuc-dropdown__group__select-menu__highlight"
+        )
+      ).to.equal(true);
+    });
+
```

Comment:
Please add test case named "should not hide menu element when clicking disabled item"